### PR TITLE
fix: jUnit dont panic evaluation if invalid yaml or k8s

### PR DESCRIPTION
**Pull request by TzlilSwimmer123**

### CODE REVIEW

**Comment by royhadad** [comment]
does this only stop the panic? or also adds something to the output?

File: bl/evaluation/junit_output.go [hunk 1]
```diff
@@ -211,3 +217,67 @@ func findFileInPolicyValidationResults(fileName string, policyValidationResults
...
+			Name: invalidYamlFile.Path,
+		}
+
+		testCase := testCase{
+			Name:      "invalid yaml file",
+			ClassName: "yaml validation",
+			Skipped:   nil,
+			Failure: &failure{
+				Message: "Invalid yaml file",
+				Content: invalidYamlFile.ValidationErrors[0].Error(),
```
Comment (TzlilSwimmer123) [inline bl/evaluation/junit_output.go]: On yaml validation failure there is only 1 error

**Comment by TzlilSwimmer123** [comment]
@royhadad it stops the panic and also adds test suit for the validations

**Comment by royhadad** [comment]
I don't see a change in the junit test file (expected output), why are the tests passing?

**Comment by TzlilSwimmer123** [comment]
@royhadad it passed until now and will keep passing because we don't test invalid files in other outputs so it's fine we will not test it on JUnit in particular. It works when testing the code manually

**Review by royhadad — APPROVED** [review]

**Pull request merged**